space.celestiaproject.Celestia.yml

```diff
@@ -0,0 +1,88 @@
+id: space.celestiaproject.Celestia
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+command: celestia-qt6
+rename-icon: celestia
+rename-desktop-file: celestia-qt6.desktop
+#rename-appdata-file: space.celestia.celestia_qt6.metainfo.xml
+finish-args:
+  - --device=dri
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=ipc
+  - --filesystem=xdg-documents/Celestia:create
+
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: '23.08'
+    no-autodownload: true
+    autodelete: false
+
+cleanup-commands:
+  - mkdir -p /app/lib/ffmpeg
```

Comment:
this doesn't belong to cleanup-commands. It belong to a `post-install`. usually the main module.